client/src/features/users/Auth/SignupNonProfit.tsx

```diff
@@ -43,84 +32,87 @@ const useStyles = makeStyles((theme: Theme) => ({
   arrow: { cursor: 'pointer' },
 }));
 
-interface SignupData {
-  org_name: string;
-  city: string;
-  state: string;
-  ein: string;
-  tax_exempt_id: string;
-  nonprofit_classification: string;
-  firstName: string;
-  last_name: string;
-  role_or_title: string;
-  email: string;
-  password: string;
-  accept_terms: boolean;
-}
-
-const initialFormData: SignupData = {
-  org_name: '',
+const defaultOrg: Organization = {
+  name: '',
+  doing_business_as: '',
   city: '',
   state: '',
   ein: '',
-  tax_exempt_id: '',
+  description: '',
+  website: '',
+  address: '',
+  phone: '',
   nonprofit_classification: '',
+};
+
+const defaultUser: BaseUserEntity = {
   firstName: '',
   last_name: '',
-  role_or_title: '',
   email: '',
-  password: '',
-  accept_terms: false,
+};
+
+const defaultUserOrg: UserOrg = {
+  approvalStatus: ApprovalStatus.pending,
+  role: Role.owner,
```

Comment:
i think if someone is signing up a new nonprofit we should just make them the org owner right off the bat. also their approvalStatus is kind of moot at that point
so i think we can remove this state and set these things by hardcoding o default on BE